.devcontainer/devcontainer.json

```diff
@@ -20,11 +20,9 @@
         }
     },
     // Set *default* container specific settings.json values on container create.
-    "settings": {
-        "python.defaultInterpreterPath": "/venv/bin/python"
-    },
     "customizations": {
         "vscode": {
+            "python.defaultInterpreterPath": "/venv/bin/python",
```

Comment:
Could you make this look like the copier template: https://github.com/DiamondLightSource/python-copier-template/blob/main/.devcontainer/devcontainer.json

Reply:
ah sure, bad nesting on my end